packages/amplication-build-manager/src/build-runner/build-runner.controller.spec.ts

```diff
@@ -139,7 +140,7 @@ describe("BuildRunnerController", () => {
     ).rejects.toThrow(errorMock);
 
     expect(mockKafkaServiceEmitMessage).toBeCalledWith(
-      configService.get(Env.CODE_GENERATION_FAILURE_TOPIC),
+      configService.get(KAFKA_TOPICS.CODE_GENERATION_FAILURE_TOPIC),
```

Comment:
https://github.com/amplication/amplication/pull/6698/files#diff-243e5dc75c8a6ca029dac34dadf96afd121d56b8c6895888240190d8a1a3f4d2R24

```suggestion
      CODE_GENERATION_FAILURE_TOPIC,
```